Formula/p/pnpm.rb

```diff
@@ -1,21 +1,21 @@
 class Pnpm < Formula
   desc "Fast, disk space efficient package manager"
   homepage "https://pnpm.io/"
-  url "https://registry.npmjs.org/pnpm/-/pnpm-9.15.3.tgz"
-  sha256 "c1da43727ccbc1ed42aff4fd6bdb4b1e91e65a818e6efff5b240fbf070ba4eaf"
+  url "https://registry.npmjs.org/pnpm/-/pnpm-10.0.0.tgz"
+  sha256 "43abf6e720fb7bc53c591b0862605c7d323d0a9d2dd33bcac07b062e13cf4948"
   license "MIT"
 
   livecheck do
-    url "https://registry.npmjs.org/pnpm/latest"
+    url "https://registry.npmjs.org/pnpm/latest-10"
```

Comment:
maybe just `latest`?

```suggestion
    url "https://registry.npmjs.org/pnpm/latest"
```

Reply:
no, latest points to pnpm@9 for now

Reply:
can you raise an upstream issue about it? (just to be sure 10 is indeed the latest release)

Reply:
Created one https://github.com/pnpm/pnpm/issues/8964

Reply:
thanks!